packages/lesswrong/server/migrations/2022-08-29-syncEmailsFields.ts

```diff
@@ -0,0 +1,38 @@
+import { registerMigration } from './migrationUtils';
+import Users from '../../lib/collections/users/collection';
+
+registerMigration({
+  name: "syncEmailsFields",
+  dateWritten: "2022-08-29",
+  idempotent: true,
+  action: async () => {
+    const usersMissingEmail = await Users.find({email: null, emails:{$exists:true}}).fetch()
+
+    // eslint-disable-next-line no-console
+    console.log(`setting emails for ${usersMissingEmail.length} users`)
+
+    let syncedUserCount = 0
+
+    let duplicateEmails: Array<string> = [] 
```

Comment:
nit: this can be `const` rather than `let`, since we aren't reassigning the value (just pushing to the existing array)